app/AppKernel.php

```diff
@@ -21,6 +21,7 @@ public function registerBundles()
             new WhiteOctober\PagerfantaBundle\WhiteOctoberPagerfantaBundle(),
             new CodeExplorerBundle\CodeExplorerBundle(),
             new AppBundle\AppBundle(),
+            new Doctrine\Bundle\FixturesBundle\DoctrineFixturesBundle(), // used for initial population of non-SQLite databases in production envs
```

Comment:
I don't think it makes sense in a "real" application, might make sense as a demo app though.